llvm/lib/Target/AMDGPU/AMDGPUInstructions.td

```diff
@@ -502,15 +502,15 @@ def zextloadi16_#as : PatFrag<(ops node:$ptr), (zextloadi16 node:$ptr)> {
   let IsLoad = 1;
 }
 
-def atomic_load_16_#as : PatFrag<(ops node:$ptr), (atomic_load_16 node:$ptr)> {
+def atomic_load_nonext_16_#as : PatFrag<(ops node:$ptr), (atomic_load_nonext_16 node:$ptr)> {
```

Comment:
Could leave the old name (`atomic_load_16_#as`) if we ever change `atomic_load_nonext_16` back to `atomic_load_16`.
Not a strong objection.

Reply:
That would be better, but requires touching more code at once